proto/minder/v1/minder.proto

```diff
@@ -2024,7 +2098,13 @@ message RuleType {
     ];
 
     // short_failure_message is the message to display when the evaluation fails.
-    string short_failure_message = 10;
+    string short_failure_message = 10 [
+        (buf.validate.field).string = {
+            pattern: "[A-Za-z][-/.!?,:;[:word:] *#\\[\\]()`{}]*",
```

Comment:
Is this markdown, or plain-text?

Reply:
It should be markdown

Reply:
Removed the pattern check but left the length one 👍